test/index.js

```diff
@@ -3330,6 +3330,66 @@ describe('list', function() {
 
 describe('object', function() {
 
+  describe('prop', function() {
+
+    it('is a binary function', function() {
+      eq(typeof S.prop, 'function');
+      eq(S.prop.length, 2);
+    });
+
+    it('type checks its arguments', function() {
+      assert.throws(function() { S.prop(1); },
+                    errorEq(TypeError,
+                            '‘prop’ expected a value of type String ' +
+                            'as its first argument; received 1'));
+
+      assert.throws(function() { S.prop('a', null); },
+                    errorEq(TypeError,
+                            '‘prop’ requires ‘a’ to implement Accessible; ' +
+                            'Null does not'));
+
+      assert.throws(function() { S.prop('a', true); },
+                    errorEq(TypeError,
+                            '‘prop’ expected object to have a property ' +
+                            'named ‘a’; true does not'));
+
+      assert.throws(function() { S.prop('a', 1); },
+                    errorEq(TypeError,
+                            '‘prop’ expected object to have a property ' +
+                            'named ‘a’; 1 does not'));
+    });
+
+    it('throws when the property is not present', function() {
+      assert.throws(function() { S.prop('map', 'abcd'); },
+                    errorEq(TypeError,
+                            '‘prop’ expected object to have a property ' +
+                            'named ‘map’; "abcd" does not'));
+
+      assert.throws(function() { S.prop('c', {a: 0, b: 1}); },
+                    errorEq(TypeError,
+                            '‘prop’ expected object to have a property ' +
+                            'named ‘c’; {"a": 0, "b": 1} does not'));
+
+      assert.throws(function() { S.prop('xxx', [1, 2, 3]); },
+                    errorEq(TypeError,
+                            '‘prop’ expected object to have a property ' +
+                            'named ‘xxx’; [1, 2, 3] does not'));
+    });
+
+    it('it returns the value of the specified object property', function() {
+      eq(S.prop('a', {a: 0, b: 1}), 0);
+      eq(S.prop('0', [1, 2, 3]), 1);
+      eq(S.prop('length', 'abc'), 3);
+      eq(S.prop('x', Object.create({x: 1, y: 2})), 1);
+    });
```

Comment:
The example I suggested is still useful, but it doesn't test what I was hoping to test since `global` *also* exists as an instance property:

```javascript
> Object.prototype.hasOwnProperty.call(/x/g, 'global')
true
```

> Although, does `Object.create` not fulfil that already? `x` and `y` live on the created object's prototype.

Good point! I didn't realize that `Object.create` worked this way. :thumbsup: